springwolf-examples/springwolf-amqp-example/src/test/resources/asyncapi.json

```diff
@@ -227,7 +227,7 @@
         },
         "name": "io.github.springwolf.examples.amqp.dtos.AnotherPayloadDto",
         "title": "AnotherPayloadDto",
-        "description": "Another payload model",
+        "description": "",
```

Comment:
The description that is shown is the description of the schema, so the logic place to show this description is on the Schema-component.
The description that is on that place, should be the one of the @AsyncMessage, in my opinion.

The reason why the description is not anymore shown, is that Swagger creates now the reference to the schema-object, whereas before it was a hardcoded reference by Springwolf itself. Due to removing that hardcoded reference, we do not receive anymore the full schema, but an already referenced schema-object on the payload, without a description.
If this is a blocking matter, what could be done is:
- Either, fetching the schema from the swagger-reference and fetching the description via that way.
- Either, setting the description on the reference-schema with a custom modelConverter.

Reply:
Good point regarding the location of the description and it seems that the description does not belong here.

Springwolf has two ways to specify an optional description via the Swagger Schema annotation (belongs to the schema) and via the AsyncOperation annotation (belongs to the operation).

Can the description be null in that case instead of an empty string?
(I am currently unsure, whether the ui selects the description already from the correct location)